service/src/main/java/bio/terra/workspace/service/policy/flight/MergePolicyAttributesUtils.java

```diff
@@ -0,0 +1,20 @@
+package bio.terra.workspace.service.policy.flight;
+
+import bio.terra.policy.model.TpsPaoGetResult;
+import bio.terra.workspace.service.policy.TpsApiDispatch;
+import java.util.Optional;
+import java.util.UUID;
+
+public class MergePolicyAttributesUtils {
+  // Since policy attributes were added later in development, not all existing
+  // workspaces have an associated policy attribute object. This method creates
+  // an empty one if it does not exist.
+  protected static void createPaoIfNotExist(TpsApiDispatch tpsApiDispatch, UUID workspaceId) {
```

Comment:
this seems very useful. 

should we just put that into TpsApiDispatch and do that for all the TpsApiDispatch method that's modifying policy on a workspace?

Reply:
That is a really good idea. I think I'll do that as part of PF-2502 so I don't keep expanding the scope of this review.